modules/navier_stokes/test/tests/finite_volume/materials/ergun/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [ergun]
```

Comment:
@lindsayad This test does not seem to be valgrind clean, and is currently impeding next -> devel merges. See https://civet.inl.gov/job/1899661/ for details.

Reply:
Looks like a timeout?

This could just be an expensive test it s kind of a legacy one.

Reply:
looking into it here
https://github.com/idaholab/moose/pull/26147

Reply:
Thanks! It finished the valgrind run before registering a timeout, with the following result:

```
test:finite_volume/materials/ergun.ergun: ==1645463== LEAK SUMMARY:
test:finite_volume/materials/ergun.ergun: ==1645463==    definitely lost: 0 bytes in 0 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==    indirectly lost: 0 bytes in 0 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==      possibly lost: 1,540 bytes in 26 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==    still reachable: 22,957,053 bytes in 111,280 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==                       of which reachable via heuristic:
test:finite_volume/materials/ergun.ergun: ==1645463==                         newarray           : 81,568 bytes in 34 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==                         multipleinheritance: 36,800 bytes in 92 blocks
test:finite_volume/materials/ergun.ergun: ==1645463==         suppressed: 0 bytes in 0 blocks
```

Reply:
Ah i didn't know there was a longer log at the bottom. 

Well even in there it looks to me that it s a timeout on step 13

Reply:
Finally got a next merge after invalidating this timeout three times :D